src/js/api/config.js

```diff
@@ -61,9 +61,39 @@ function _adjustDefaultBwEstimate(estimate) {
     return Defaults.bandwidthEstimate;
 }
 
+function _copyToLocalization(allOptions) {
+    const { advertising, related, sharing } = allOptions;
+    const localization = Object.assign({}, allOptions.localization);
+
+    if (advertising) {
+        localization.advertising = Object.assign({}, localization.advertising, advertising);
+    }
+
+    if (related) {
+        localization.related = localization.related || {};
+        if (typeof related === 'object') {
+            localization.related.heading = localization.related.heading || related.heading;
```

Comment:
there is no `related.heading` in the documentation, why do we fallback to it ? https://developer.jwplayer.com/jw-player/docs/developer-guide/customization/configuration-reference/